### PR TITLE
Get rid of chef_gem

### PR DESCRIPTION
**Pull request by josephholsten**
fix the middle dot in #21

### CODE REVIEW

File: recipes/replicaset.rb [hunk 1]
```diff
@@ -21,7 +21,15 @@
 node.set['mongodb']['cluster_name'] = node['mongodb']['cluster_name']
 
 include_recipe 'mongodb::install'
-include_recipe 'mongodb::mongo_gem'
+
+ruby_block 'chef_gem_at_converge_time' do·
```
Comment (triccardi-systran) [inline recipes/replicaset.rb]: There was a second one...
This code could be factorized, to avoid such issue.

**Comment by luishdez** [comment]
wt… another dot … 

@josephholsten all PRs are branches within this repo, you can checkout pr/370 and commit there to avoid duplicated PRs. I'm closing the original PR

File: recipes/replicaset.rb (hunk 1 above)
Comment (zarry) [inline recipes/replicaset.rb]: I am almost less worried about that right now and more interested in getting this fix in.  Should that hold getting this out?

**Comment by zarry** [comment]
Aside from the slight duplication anything else stopping a merge on this?

**Comment by triccardi-systran** [comment]
I thought the middle dot should break everything, I don't understand why CI raises no issue.

The code duplication is minor and should not be blocking.

**Comment by zarry** [comment]
I removed the dot, once travis gives us the OK I will merge.

**Pull request merged**